python/whl.bzl

```diff
@@ -36,6 +36,9 @@ def _extract_wheels(ctx, wheels):
         args += ["--add-build-content=%s" % ctx.path(ctx.attr.additional_build_content)]
     args += ["--extras=%s" % extra for extra in ctx.attr.extras]
 
+    # Add our sitecustomize.py that ensures all .pth files are run.
+    args += ["--add-dependency=@io_bazel_rules_python//python:site"]
```

Comment:
In theory yes, but it would involve a fair amount of plumbing since we have a fair amount or indirection here. :)

Also it seems that rules projects casually use the repo name, at least in the the rule definitions:
https://github.com/bazelbuild/rules_go/blob/master/go/def.bzl#L15
https://github.com/bazelbuild/rules_docker/blob/master/container/image.bzl#L506
https://github.com/bazelbuild/rules_docker/blob/master/go/image.bzl#L32